frontend/src/views/activity/Activity.vue

```diff
@@ -156,6 +156,13 @@ export default {
     PdfDownloadButtonNuxt
   },
   mixins: [campRoleMixin],
+  provide () {
+    return {
+      preferredContentTypes: () => this.preferredContentTypes,
+      rootContentNodes: () => this.contentNodes,
```

Comment:
I like it this way :+1:
Only the name `rootContentNodes` is not optimal I think. When I read that in a component which gets it injected, I think of a set of root content nodes, i.e. the roots of multiple trees. How about `allContentNodes` or `contentNodesInTree`?

Reply:
Renamed to `allContentNodes` in https://github.com/ecamp/ecamp3/pull/2683/commits/eab7b427f9c1c8819aead2d0d4f3cea912773cc9

Reply:
Much better name!